plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/glue/TestIcebergGlueCatalogAccessOperations.java

```diff
@@ -545,8 +545,7 @@ public void testSystemMetadataTableComments()
                             session,
                             "SELECT * FROM system.metadata.table_comments WHERE schema_name = CURRENT_SCHEMA AND table_name LIKE 'test_select_s_m_t_comments%'",
                             ImmutableMultiset.<GlueMetastoreMethod>builder()
-                                    .addCopies(GET_TABLES, 3)
-                                    .addCopies(GET_TABLE, tables * 2)
+                                    .addCopies(GET_TABLES, 1)
```

Comment:
```diff
- .addCopies(GET_TABLES, 3)
- .addCopies(GET_TABLE, tables * 2)
+ .addCopies(GET_TABLES, 1)
```

looks like a nice change. thank you @ebyhr for your hint https://github.com/trinodb/trino/pull/18315#pullrequestreview-1556000306
cc @kokosing @atanasenko @alexjo2144